backend/windmill-worker/src/worker.rs

```diff
@@ -1481,6 +1481,25 @@ fn capitalize(s: &str) -> String {
     }
 }
 
+fn get_common_deno_proc_envs(token: &str, base_internal_url: &str) -> HashMap<String, String> {
+    let hostname_base = BASE_URL.split("://").last().unwrap_or("localhost");
+    let hostname_internal = base_internal_url.split("://").last().unwrap_or("localhost");
+    let deno_auth_tokens_base = DENO_AUTH_TOKENS.as_str();
+    let deno_auth_tokens =
+        format!("{token}@{hostname_base};{token}@{hostname_internal}{deno_auth_tokens_base}",);
+
+    let mut deno_envs: HashMap<String, String> = HashMap::new();
+    deno_envs.insert(String::from("PATH"), PATH_ENV.clone());
+    deno_envs.insert(String::from("DENO_AUTH_TOKENS"), deno_auth_tokens);
+    deno_envs.insert(String::from("BASE_INTERNAL_URL"), base_internal_url.to_string());
+    deno_envs.insert(String::from("NO_COLOR"), String::from("true"));
```

Comment:
nit but I think the initialize style might be a tiny bit better:
```
let vikings = HashMap::from([
    ("Norway", 25),
    ("Denmark", 24),
    ("Iceland", 12),
]);
```

Reply:
Thanks, I know my `rust` skills are pretty lacking right now, trying to learn still.  I can clean this up.